gitpage/app/views/gitpagedisplay/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>GitHub</h1>
+
+<div id="right">
+    <nav>
+        <div>Overview</div>
+        <div>Repositories</div>
+        <div><%= @profile['public_repos'] %></div>
+    </nav>
+
+    <div id="repos">
+        <% @repos.each do |repo| %>
+            <div id="title"><%= repo['name'] %><br></div>
+            <%= repo['description'] %><br>
+        <% end %>
+    </div>
+</div>
```

Comment:
Yeah, without data it's hard to structure the page. One way to make progress when you are blocked on another piece is to hard-code what you're missing and work around that problem.

Reply:
Also! This view is linked in the Rails world to the matching controller. You have your get request to the API in a controller named `GitpageController`, but if you move it into `GitpagedisplayController#index` it will work!